react-native-radar.podspec

```diff
@@ -15,5 +15,5 @@ Pod::Spec.new do |s|
   s.platform = :ios, "10.0"
 
   s.dependency "React"
-  s.dependency "RadarSDK", "3.1.7"
+  s.dependency "RadarSDK", "3.1.10"
```

Comment:
not `3.1.14`, like in `android/build.gradle`?

Reply:
Nope, iOS is only on `3.1.10`